site/cypress/e2e/Cypress-TAGrading/ta_grading_details.spec.js

```diff
@@ -2,19 +2,14 @@ describe('TA Grading details page', () => {
     it('ta grading ui testing', () => {
         cy.login();
         cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
-        cy.get('[data-testid="view-sections"]').should('contain', 'View All');
+        cy.get('[data-testid="view-your-sections"]').should('contain', 'Only Assigned Sections');
         cy.get('[data-testid="view-sections"]').click();
-        cy.get('[data-testid="view-sections"]').should('contain', 'View Your Sections');
-        cy.get('[data-testid="random-default-order"]').should('contain', 'Switch to Random Order');
-        cy.get('[data-testid="random-default-order"]').click();
-        cy.get('[data-testid="random-default-order"]').should('contain', 'Switch to Default Order');
-        cy.get('[data-testid="toggle-anon-button"]').should('contain', 'Enable Anonymous Mode');
-        cy.get('[data-testid="toggle-anon-button"]').click();
-        cy.get('[data-testid="toggle-anon-button"]').should('contain', 'Disable Anonymous Mode');
-        cy.get('[data-testid="toggle-grade-inquiry"]').should('contain', 'Grade Inquiry Only: Off');
-        cy.get('[data-testid="toggle-grade-inquiry"]').click();
-        cy.get('[data-testid="toggle-grade-inquiry"]').should('contain', 'Grade Inquiry Only: On');
-        cy.get('[data-testid="toggle-grade-inquiry"]').click();
+        cy.get('[data-testid="random-default-order"]').should('contain', 'Randomize Order');
+
+        cy.get('[data-testid="toggle-anon-button"]').should('contain', 'Anonymize Student Name');
+
+        cy.get('[data-testid="toggle-grade-inquiry"]').should('contain', 'Only Grade Inquiry');
+
```

Comment:
Let's match the formatting of the surrounding code:

```suggestion
        cy.get('[data-testid="random-default-order"]').should('contain', 'Randomize Order');
        cy.get('[data-testid="toggle-anon-button"]').should('contain', 'Anonymize Student Name');
        cy.get('[data-testid="toggle-grade-inquiry"]').should('contain', 'Only Grade Inquiry');
```